book/dataframes.md

```diff
@@ -88,58 +87,57 @@ We can have a look at the first lines of the file using [`first`](/commands/docs
 
 ...and finally, we can get an idea of the inferred data types:
 
-```nu no-run
-> $df | polars dtypes
-╭───┬───────────┬───────╮
-│ # │  column   │ dtype │
-├───┼───────────┼───────┤
-│ 0 │ anzsic06  │ str   │
-│ 1 │ Area      │ str   │
-│ 2 │ year      │ i64   │
-│ 3 │ geo_count │ i64   │
-│ 4 │ ec_count  │ i64   │
-╰───┴───────────┴───────╯
+```nu
+> $df | polars schema
+╭───────────┬─────╮
+│ anzsic06  │ str │
+│ Area      │ str │
+│ year      │ i64 │
+│ geo_count │ i64 │
+│ ec_count  │ i64 │
+╰───────────┴─────╯
 ```
 
 ### Loading the file
 
 Let's start by comparing loading times between the various methods. First, we
 will load the data using Nushell's [`open`](/commands/docs/open.md) command:
 
-```nu no-run
-> timeit {open .\Data7602DescendingYearOrder.csv}
-30sec 479ms 614us 400ns
+```nu
+> timeit {open Data7602DescendingYearOrder.csv}
+1sec 705ms 555µs 208ns
 ```
 
-Loading the file using native Nushell functionality took 30 seconds. Not bad for
+Loading the file using native Nushell functionality took 1.63 seconds. Not bad for
 loading five million records! But we can do a bit better than that.
 
 Let's now use Pandas. We are going to use the next script to load the file:
 
-```python
-import pandas as pd
+```nu
+('import pandas as pd
 
-df = pd.read_csv("Data7602DescendingYearOrder.csv")
+df = pd.read_csv("Data7602DescendingYearOrder.csv")'
+| save load.py -f)
 ```
 
 And the benchmark for it is:
 
-```nu no-run
+```nu
 > timeit {python load.py}
-2sec 91ms 872us 900ns
+3sec 539ms 303µs 417ns
 ```
 
-That is a great improvement, from 30 seconds to 2 seconds. Nicely done, Pandas!
+Here bare nushell goes almost like pandas!
 
 Probably we can load the data a bit faster. This time we will use Nushell's
 `polars open` command:
 
-```nu no-run
-> timeit {polars open .\Data7602DescendingYearOrder.csv}
-601ms 700us 700ns
+```nu
+> timeit {polars open Data7602DescendingYearOrder.csv | polars collect; null}
+11sec 994ms 700µs 125ns
 ```
 
-This time it took us 0.6 seconds. Not bad at all.
+This time it took us 0.00053 seconds. Not bad at all.
```

Comment:
where did this time come from? above it says 11 sec, which I don't understand either. I'm not sure how dataframes would be slower than nushell opening the same file.

Reply:
11 seconds is the execution time inside numd script. I executed it manually now and it shows 5 seconds which is also above expectations. 

0.00053 seconds is the time it took to create a lazyframe plan. I will fix it in the next commit.